identity-service/src/relay/txRelay.js

```diff
@@ -139,23 +145,61 @@ const sendTransactionInternal = async (req, web3, txProps, reqBodySHA) => {
       `L2 - txRelay - selected wallet ${wallet.publicKey} for sender ${senderAddress}`
     )
 
-    if (sendToNethermind) {
-      const ok = await relayToNethermind(encodedABI)
-      txParams = ok.txParams
-      txReceipt = ok.receipt
-    } else {
-      // use POA receipt as main receipt
-      const ok = await createAndSendTransaction(
-        wallet,
-        contractAddress,
-        '0x00',
-        web3,
-        req.logger,
-        gasLimit,
-        encodedABI
+    // send to POA
+    // PROD doesn't have sendToNethermindOnly and should default to POA
+    // STAGE defaults to nethermind but can send to POA when it has both addresses
+    const relayPromises = []
+    if (
+      !sendToNethermindOnly ||
+      (sendToNethermindOnly && nethermindContractAddress)
```

Comment:
this boolean logic looks suss to me... wonder if we can simplify the conditions a bit... want to discuss on hangout?

Reply:
yeaaa i was tryna think of ways to make it simpler